runtime/types.ts

```diff
@@ -9,6 +9,7 @@ export enum FormulaType {
   Sync = 'Sync',
   SyncUpdate = 'SyncUpdate',
   GetPermissions = 'GetPermissions',
+  ExecuteGetPermissions = 'ExecuteGetPermissions',
```

Comment:
(I think things are still in-flight here and this is on your radar, but in the final version we'd want to revert adding a new FormulaType, and only rely on contextOptions for the new behavior.)